res/quad.fs.glsl

```diff
@@ -8,11 +8,16 @@ bool PointInRect(vec2 p, vec2 p0, vec2 p1)
 
 void main(void)
 {
-    // Clip out rect
+    // Clip out.
     if (PointInRect(vPosition, vClipOutRect.xy, vClipOutRect.zw)) {
         discard;
     }
 
+    // Clip in.
+    if (vClipInRect != vec4(0.0) && !PointInRect(vPosition, vClipInRect.xy, vClipInRect.zw)) {
```

Comment:
Oh I see what's happening - perhaps just set it to a large value in the VS to avoid a conditional in the FS?